pom.xml

```diff
@@ -65,6 +65,19 @@
                     </execution>
                 </executions>
             </plugin>
+            <plugin>
+                <groupId>org.apache.maven.plugins</groupId>
+                <artifactId>maven-jar-plugin</artifactId>
+                <version>3.1.0</version>
+                <configuration>
+                    <archive>
+                        <manifest>
+                            <addClasspath>true</addClasspath>
+                            <mainClass>com.oleeo.di.tika.GherkinServerCli</mainClass>
```

Comment:
I think this should be removed (the `<mainClass>` section or the entire `<configuration>` section).

Reply:
Thanks for catching this. I've removed the configuration part.